cmd/snap/main.go

```diff
@@ -92,6 +94,17 @@ type parserSetter interface {
 // Since commands have local state a fresh parser is required to isolate tests
 // from each other.
 func Parser() *flags.Parser {
+	optionsData.Version = func() error {
+		cv, err := Client().Version()
+		if err != nil {
+			cv = i18n.G("unavailable")
+		}
+
+		fmt.Fprintf(Stdout, i18n.G("This is snap %s; snapd %s.\n"), cmd.Version(), cv)
```

Comment:
Can we please make it slightly more tool friendly. E.g.:

snap: ...
snapd: ...